### PR TITLE
fix(combobox/text-input): add svg top positioning for IE11

### PR DESCRIPTION
**Pull request by dakahn**
partially addresses #5335

Adds `top` attribute to SVG positioning in text-input and combobox getting that SVG sitting properly in the form element. Also adds the css-reset to combobox so that when #5357 lands the text-input will `display: none` for `::-ms-clear` getting rid of that big funky browser default **X** 👍 

#### Testing / Reviewing

check the clear button positioning in IE11 for both text-input and combobox. Then check the clear button positioning in modern browsers just to be safe 🦅 

#### _Note_

I didn't stick the top attribute into a query for IE11 specifically because adding the `top` attribute had no negative effect on Firefox and Chrome. If that wont fly let me know 💟

### CODE REVIEW

**Comment by laurenmrice** [comment]
I checked all the other browsers just in case and they look good. Will see if I can check ie11 with josh today (since I can't view on my comp)

File: packages/components/src/components/list-box/_list-box.scss [hunk 1]
```diff
@@ -343,6 +343,7 @@ $list-box-menu-width: rem(300px);
   .#{$prefix}--list-box__selection {
     position: absolute;
     right: rem(33px); // to preserve .5rem space between icons according to spec
+    top: 0.35rem;
```
Comment (joshblack) [inline packages/components/src/components/list-box/_list-box.scss]: Do we need to use bespoke positioning or could we do `top: 0; bottom: 0;` to center?

Comment (dakahn) [reply packages/components/src/components/list-box/_list-box.scss]: let me check 👍

Comment (joshblack) [reply packages/components/src/components/list-box/_list-box.scss]: Seems like we settled on `top: 50%;` `bottom: translateY(-50%);` 👍

**Review by laurenmrice — APPROVED** [review]

**Review by joshblack — APPROVED** [review]
Seems great once text input is updated!

**Review by asudoh — APPROVED** [review]
LGTM 👍 - Thanks @dakahn!

**Comment by emyarod** [comment]
this causes a regression for the inline multiselect

**Comment by emyarod** [comment]
this causes a regression for multiselects

image

**Comment by asudoh** [comment]
Oh looks a good catch - Have you reported it @emyarod?

**Pull request merged**